news/editions/edition3.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: single
+title:  "*Flying*  Edition 3"
+sidebar:
+  nav: news
+redirect_from: /news/latest/
+---
+## Open Source: #BetterTogether
+> This month's issue focuses Spinnaker's strong open source DNA. The OSS community embraces Spinnaker, as it builds a bridge between the collaboration and interoperability story at the core of Linux culture, and the powerful cloud economy that has leveraged it to transform software delivery. Efforts to build native Kubernetes integration highlight this unique strength.
```

Comment:
> The OSS community embraces Spinnaker, as it builds a bridge between the collaboration and interoperability story at the core of Linux culture, and the powerful cloud economy that has leveraged it to transform software delivery.

^ is a pretty long sentence. The longer a sentence gets, the more meaning is lost, especially for people whose second language is English.

Also, the comma between `culture` and `and` isn't needed.